### PR TITLE
Attributes of User class

### PR DESCRIPTION
**Pull request by SallyHabib**

### CODE REVIEW

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 1]
```diff
@@ -1,5 +1,100 @@
 package org.ieeeguc.ieeeguc.models;
 
+import java.util.Date;
 public class User {
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: Add The attribute id of type int

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 2]
```diff
@@ -1,5 +1,100 @@
 package org.ieeeguc.ieeeguc.models;
 
+import java.util.Date;
 public class User {
+    private Enum type;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: Create a separate file for the Enum Type and use its datatype here

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 3]
```diff
@@ -1,5 +1,100 @@
 package org.ieeeguc.ieeeguc.models;
 
+import java.util.Date;
 public class User {
+    private Enum type;
+private  String firstName;
+    private  String LastName;
+    private String email;
+    private Enum gender;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: Create a separate file for the Enum Gender and use its datatype here

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 4]
```diff
@@ -1,5 +1,100 @@
...
+    private Enum type;
+private  String firstName;
+    private  String LastName;
+    private String email;
+    private Enum gender;
+    private Date birthdate;
+    private String ieeeMembershipID;
+    private int committeeID;
+    private String committeeName;
+
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: add a : 
private JSONObject settings;

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 5]
```diff
@@ -1,5 +1,100 @@
 package org.ieeeguc.ieeeguc.models;
 
+import java.util.Date;
 public class User {
+    private Enum type;
+private  String firstName;
+    private  String LastName;
+    private String email;
+    private Enum gender;
+    private Date birthdate;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: add a:
private String phoneNumber;

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 6]
```diff
@@ -1,5 +1,100 @@
...
+private  String firstName;
+    private  String LastName;
+    private String email;
+    private Enum gender;
+    private Date birthdate;
+    private String ieeeMembershipID;
+    private int committeeID;
+    private String committeeName;
+
+    public User(Enum type, String firstName, String lastName, String email, Enum gender, Date birthdate, String ieeeMembershipID,
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: change the constructor according to the added attributes

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 7]
```diff
@@ -1,5 +1,100 @@
...
+    }
+    public Enum getType() {
+        return type;
+    }
+
+    public void setType(Enum type) {
+        this.type = type;
+    }
+
+    public String getFirstName() {
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: remove all the setters as the attributes will be changed within this class only.
add the getters for the newly added attributes

**Review by KEMDAK — CHANGES_REQUESTED** [review]
Sorry a lot of the changes i didn't tell you about it before. :(

File: app/src/main/java/org/ieeeguc/ieeeguc/models/Gender.java [hunk 8]
```diff
@@ -0,0 +1,9 @@
+package org.ieeeguc.ieeeguc.models;
+
+/**
+ * Created by sally on 31/01/2017.
+ */
+
+public enum Gender {
+    Male,Female;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/Gender.java]: Capitalize all the letters (Java convention for Enums)

File: app/src/main/java/org/ieeeguc/ieeeguc/models/type.java [hunk 9]
```diff
@@ -0,0 +1,11 @@
+package org.ieeeguc.ieeeguc.models;
+
+/**
+ * Created by sally on 31/01/2017.
+ */
+
+public enum type {
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/type.java]: change the enum name to (Type)

File: app/src/main/java/org/ieeeguc/ieeeguc/models/type.java [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+    Member , admin , systemAdmin;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/type.java]: Capitalize all letters and for the please read the documentation carefully for the account types

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 11]
```diff
@@ -1,5 +1,102 @@
 package org.ieeeguc.ieeeguc.models;
 
+import org.json.JSONObject;
+
+import java.util.Date;
 public class User {
+    private  type Type;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: The type should start with capital however the variable name starts with a small letter

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 12]
```diff
@@ -1,5 +1,102 @@
 package org.ieeeguc.ieeeguc.models;
 
+import org.json.JSONObject;
+
+import java.util.Date;
 public class User {
+    private  type Type;
+private  String firstName;
+    private  String LastName;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: change the name of the variable to (lastName)

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java [hunk 13]
```diff
@@ -1,5 +1,102 @@
...
+    private Date birthdate;
+    private String ieeeMembershipID;
+    private int committeeID;
+    private String committeeName;
+    private int id;
+    private String phoneNumber;
+    private JSONObject settings;
+
+
+    public User( type Type, String firstName, String lastName, String email, Gender gender, Date birthdate, String ieeeMembershipID,
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: add all the attributes to the cunstructor

**Review by KEMDAK — CHANGES_REQUESTED** [review]
There is a file named Sally delete it. and make the requested changes then push again

File: app/src/main/java/org/ieeeguc/ieeeguc/models/User.java (hunk 13 above)
Comment (SallyHabib) [reply app/src/main/java/org/ieeeguc/ieeeguc/models/User.java]: All the attributes? In the previous comment u specify some attributes

**Review by KEMDAK — APPROVED** [review]

**Pull request merged**